client.go

```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent("Heroku app: "+*data.app, v)
```

Comment:
Maybe add some way how to identify the app in question like  <APP_NAME> from `Oct 09 18:26:54 <APP_NAME> heroku/api:  Scale to web=6 by user@company.com`